### DISCUSSION TITLE
Build error : utput conflict: multiple input files are writing to `dist/Scriptorium/index.html`. Use distinct `permalink` values to resolve this conflict.

### DISCUSSION THREAD

**Discussion by Mara-Li** [oleeskild/obsidian-digital-garden#257] [Q&A]
I dunno why, but multiple different file doesn't want to build: 
![image](https://user-images.githubusercontent.com/30244939/227791027-1af26f34-3e90-4f3d-b718-18bbdaac9430.png)

Repo : https://github.com/Lisandra-dev/mara-garden/

Anyone have an idea ? :s

**Comment by uroybd** [comment]
The message is clear. You have 2 or more files using the same permalink `dist/Scriptorium/index.html`.

**Reply by Mara-Li** [reply]
But, I don't have any settings for date in any file :/

**Reply by uroybd** [reply]
It has nothing to do with dates. All 5 files listed here are resolving to the same URL, either through `dg-path` or `dg-permalink`. The repo you've shared seems to be restricted so can't tell exactly what is wrong.
